File: mybatis-plus-join-test/test-join/src/main/java/com/github/leheyue/test/join/service/impl/UserServiceImpl.java
package com.github.leheyue.test.join.service.impl;

import com.github.leheyue.base.MPJBaseServiceImpl;
import com.github.leheyue.test.join.entity.UserDO;
import com.github.leheyue.test.join.mapper.UserMapper;
import com.github.leheyue.test.join.service.UserService;
import org.springframework.stereotype.Service;

@Service
public class UserServiceImpl extends MPJBaseServiceImpl<UserMapper, UserDO> implements UserService {


}

File: mybatis-plus-join-test/test-kotlin/src/main/kotlin/com/github/yulichang/test/kt/service/impl/UserServiceImpl.kt
package com.github.leheyue.test.kt.service.impl

import com.github.leheyue.base.MPJBaseServiceImpl
import com.github.leheyue.test.kt.entity.UserDO
import com.github.leheyue.test.kt.mapper.UserMapper
import com.github.leheyue.test.kt.service.UserService
import org.springframework.stereotype.Service

@Service
open class UserServiceImpl : MPJBaseServiceImpl<UserMapper?, UserDO?>(), UserService
